### ISSUE TITLE
Adding classes to <code> in fenced code blocks

### ISSUE THREAD

**Issue by felixsanz** [arve0/markdown-it-attrs#13]
How i can add classes to `<code>` tags when using fenced code blocks?

This doesn't work:

    ``` {.foo}
    text
    ```

**Comment by arve0** [comment]
Use this syntax:

<pre>
```python
for i in range(10):
    print(i)
```
</pre>

Define the [class prefix in markdown-it](https://github.com/markdown-it/markdown-it#init-with-presets-and-options).

**Comment by arve0** [comment]
[Output](https://jsfiddle.net/ge0rqw46/):
```
<pre>
<code class="language-python">for i in range(10):
    print(i)
</code>
</pre>
```

**Comment by felixsanz** [comment]
But i want to add classes, not the language at all.

`<pre><code class="foo">`

**Comment by arve0** [comment]
I see. That is not implemented. 

> This is the purpose of this plugin, isn't it?

Yes.

**Comment by arve0** [comment]
Let me know how #14 works for you, and I'll merge and release to npm.

**Comment by felixsanz** [comment]
awesome! thanks for the quick fix

**Comment by arve0** [comment]
Released in v0.4.0.
